### ISSUE TITLE
SimulatedNetworks and Switches are ignored in Ethernet Demo

### ISSUE THREAD

**Issue by fgrzeszc** [vectorgrp/sil-kit#96]
Hello,

I pulled the latest version from master and tried to run Ethernet demo with NetworkSimulatorConfig.yaml

`./Release/SilKitDemoEthernet  ../Demos/Ethernet/NetworkSimulatorConfig.yaml EthernetWriter
SIL Kit Version: 4.0.52
YAML validation returned warnings:
At line 2, column 0: Element "SimulatedNetworks" is being ignored. It is not a sub-element of schema path "/"
At line 9, column 0: Element "Switches" is being ignored. It is not a sub-element of schema path "/"
`

It seems it is not parsed / supported, at least with code in the demo.
Does it require creating own implementation using NetworkSimulator API?

Regards
Filip

**Comment by VDanielEdwards** [comment]
Hello @fgrzeszc! Thank you for your message.

TL;DR: You can safely ignore these.

As you have already noticed, all the configuration files named `NetworkSimulatorConfig.yaml` are not SIL Kit configuration files. They use a separate schema for the proprietary network simulator, which is not publically available - and are primarily used for internal testing.

We might opt for removing these files from the repository, as I can see that they can create some confusion.

The network simulator API allows you to implement a network simulator of your own, you are not bound to these specific configuration files.

I'll close this issue for now, but please feel free to re-open it if you have further questions.
